chemclipse/plugins/org.eclipse.chemclipse.chromatogram.msd.peak.detector.fragment.test/src/org/eclipse/chemclipse/chromatogram/msd/peak/detector/settings/PeakDetectorSettings.java

```diff
@@ -11,10 +11,30 @@
  *******************************************************************************/
 package org.eclipse.chemclipse.chromatogram.msd.peak.detector.settings;
 
+import java.util.Collection;
+import java.util.Collections;
+
+import org.eclipse.chemclipse.numeric.statistics.WindowSize;
+
 /**
  * THIS IS A TESTCLASS! ONLY USE IT FOR TEST CASES! IT'S NOT FOR PRODUCTIVE USE!
```

Comment:
The comment should be updated, this seems to be strange, why is there a special class for test cases only? why is it then not included in the test-bundle but here?

Reply:
As far as I can see this is actually located in the test-bundle.
Resolved with 26855ee76180426d5f9f234cb1df9e0cb53fbc96.